tests/move/transfer/sources/script_coin_transfer.move

```diff
@@ -0,0 +1,12 @@
+script {
```

Comment:
can we reuse https://github.com/aptos-labs/aptos-ts-sdk/blob/main/tests/move/transfer/sources/script_transfer.move instead of creating another move script test file?

Reply:
We can, though I wanted to ensure there was an example with and without a type argument